src/main/java/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -749,4 +753,21 @@ public static InvalidEntityException convertParsingError(Throwable throwable) {
         }
         return new InvalidEntityException("Error parsing JSON in request body: " + throwable.getMessage());
     }
+
+    /**
+     * Make a session web cookie based on given sessonToken and expireInSeconds (duration).
+     *
+     * @param sessionToken the sessionToken of the cookie
+     * @param expireInSeconds the effective duration of this cookie
+     * @return the Cookie
+     */
+    public static Cookie makeSessionCookie(String sessionToken, int expireInSeconds) {
+        Cookie cookie = new Cookie(SESSION_TOKEN_HEADER, sessionToken);
+        cookie.setMaxAge(expireInSeconds);
+        cookie.setPath("/");
+        cookie.setDomain("localhost");
+        cookie.setHttpOnly(false);
+        cookie.setSecure(false);
+        return cookie;
+    }
```

Comment:
Consider putting this in HttpUtil in the org.sagebionetworks.bridge.spring.util, so we can keep our Bridge stuff separate from our HTTP stuff.